examples/fungible/src/contract.rs

```diff
@@ -294,7 +294,7 @@ impl FungibleToken {
                 owner: account.owner,
                 amount,
             };
-            ExecutionResult::default().with_message(account.chain_id, message)
```

Comment:
So if this is a `Credit` from chain A with owner Alice to chain B with owner Bob, it will bounce and be credited to Bob on chain A. That seems okay-ish, but maybe it would be better to credit it to Alice and consider the transaction to be reverted? Otherwise Alice now has no way to retry sending Bob's money to chain B.

And if it was a `Withdraw`, where Alice tried to move her own money from chain B to chain C, it will now instead be moved from chain A to chain C.

Reply:
`Withdraw` is not tracked I think. You're right about `Credit`.